specs/AppLifecycle/StateNotifications/AppLifecycle StateNotifications.md

```diff
@@ -239,147 +233,142 @@ The well-known
+| GUID_BATTERY_PERCENTAGE_REMAINING | The remaining battery capacity has changed.                  | `RemainingChargePercent`, `RemainingChargePercentChanged` |
+| GUID_CONSOLE_DISPLAY_STATE        | The current monitor's display state has change. Apps should stop rendering graphics content when the monitor is off to reduce system power consumption. | _Not implemented_                                         |
+| GUID_GLOBAL_USER_PRESENCE         | The user status associated with any session has changed. This notification is only to services and other programs running in session 0. User-mode applications should register for GUID_SESSION_USER_PRESENCE instead. | _Not implemented_                                         |
+| GUID_IDLE_BACKGROUND_TASK         | The system is busy: now is a good time for apps to perform background or idle tasks that would otherwise prevent the computer from entering an idle state. | `SystemIdleStatus, SystemIdleStatusChanged`               |
+| GUID_MONITOR_POWER_ON             | The monitor has been powered on or off. From Windows 8 and Windows Server 2012, apps should use GUID_CONSOLE_DISPLAY_STATE instead. | _Not implemented_                                         |
+| GUID_POWER_SAVING_STATUS          | Battery saver has been turned on or off.                     | `EnergySaverStatus, EnergySaverStatusChanged`             |
+| GUID_POWERSCHEME_PERSONALITY      | The active power scheme personality has changed. All power schemes map to one of these personalities. | _Not implemented_                                         |
+| GUID_SESSION_DISPLAY_STATUS       | The display associated with the application's session has been powered on or off. | `DisplayStatus, DisplayStatusChanged`                     |
+| GUID_SESSION_USER_PRESENCE        | The user status associated with the application's session has changed. | `UserPresenceStatus`                                      |
+| GUID_SYSTEM_AWAYMODE              | The system is entering or exiting away mode.                 | _Not implemented_                                         |
+
+In addition, the PowerRegisterForEffectivePowerModeNotifications API is brought forward with a new property and event:
+
+| Existing API                                             | Description                             | New property and event                          |
+| -------------------------------------------------------- | --------------------------------------- | ----------------------------------------------- |
+| PowerRegisterForEffectivePowerModeNotifications function | The effective power mode of the system. | `EffectivePowerMode, EffectivePowerModeChanged` |
 
 The existing PowerManager class exposes only static properties and events: all members are valid to
 be brought over to the Reunion version. Many existing enumerated values are brought over as-is as
 well.
 
 ```idl
-namespace Microsoft.Windows.System.Power
+namespace Microsoft.ProjectReunion
```

Comment:
I think we might have to revert this once Jon is back. We will be moving forward with Windows.System.Power so we can keep this.

Reply:
Fixed